test/unit/background/datastore-test.js

```diff
@@ -23,6 +23,53 @@ const LOGINS_EVENTS = [
   "AllRemoved",
 ].map(name => `on${name}`);
 
+const cmpAlphaBy = name => (a, b) => a[name].localeCompare(b[name]);
```

Comment:
I'd figured things would be more readable by leaving this stuff toward the end of the script. Looks like this is a vote against? :)